### ISSUE TITLE
--gene_norm 'None'

### ISSUE THREAD

**Issue by f4bry** [rmarkello/abagen#152]
Running the following code:

`abagen --donors '10021' '12876' --data_dir '/home/abagen-data/' --gene_norm 'None'  --atlas_info 'atlas-desikankilliany.csv' 'atlas-desikankilliany.nii.gz'`

generates this error:

`abagen: error: argument --gene_norm/--gene-norm: invalid choice: None (choose from 'center', 'demean', 'minmax', 'mixed_sig', 'mixed_sigmoid', 'robust_sigmoid', 'rs', 'rsig', 'scaled_robust_sigmoid', 'scaled_rsig', 'scaled_sig', 'scaled_sig_qnt', 'scaled_sigmoid', 'scaled_sigmoid_quantiles', 'sig', 'sigmoid', 'srs', 'zscore', 'None')
`

**Comment by rmarkello** [comment]
Hi @f4bry ! :wave:

Thanks for raising this issue! This is definitely a bug, which I've opened #153 to address. Once the tests pass and the PR is merged in I'll make a quick bug-fix release and push the new release PyPi so you'll just have to `pip install --upgrade abagen`. I'll ping here when it's ready for the update—should only be an hour or so!

As an alternative if you are willing to use the Python API, the following should do the same thing as the code you ran:

```python
import abagen
atlas = abagen.fetch_desikan_killiany()
expression = abagen.get_expression_data(atlas['image'], atlas['info'], donors=['10021', '12876'], data_dir='/home/abagen-data', gene_norm=None)
expression.to_csv('abagen_expression.csv')
```

**Comment by rmarkello** [comment]
Alright, the new version is now released on [PyPi](https://pypi.org/project/abagen/)! You should be able to use `pip install --upgrade abagen` to update your installation and then the command you previously used should work 🤞 

I'm going to go ahead and close this issue for now, but if you run into the same problem please let me know (or open a new issue if something else seems amiss!).
